github/event_types.go

```diff
@@ -276,6 +276,27 @@ type MilestoneEvent struct {
 	Org     *Organization `json:"organization,omitempty"`
 }
 
+// OrganizationEvent is Triggered when a user is added, removed, or invited to an Organization.
+// Events of this type are not visible in timelines. These events are only used to trigger organization hooks.
+// Webhook event name is "organization"
```

Comment:
Nitpick, missing period at end of sentence. Here, and in many of the places below.